images/macos/scripts/build/configure-system.sh

```diff
@@ -11,7 +11,8 @@ close_finder_window
 
 # Remove Parallels Desktop
 # https://github.com/actions/runner-images/issues/6105
-if is_Monterey; then
+# https://github.com/actions/runner-images/issues/10143
+if is_Monterey || is_SonomaX64; then
```

Comment:
If macos12 and macos14 both support Parallels is there any reason to exclude macos13?

Reply:
Added changes for both macOS13 and macOS14